python/protoc-gen-mypy

```diff
@@ -91,9 +91,7 @@ class PkgWriter(object):
                 remains = ".".join(split[i + 1:])
                 if not remains:
                     return import_name
-                raise AssertionError("Don't support nested imports yet")
-                # return new_nested_import(import_name, remains)
-
+                return import_name + "." + remains
```

Comment:
Could you please leave a comment somewhere around here explaining what "remains" refers to - to make it clear to a peruser that it refers to a direct import of a nested enum or message from another package.

Reply:
@nipunn1313 , done.